samples/variables.go

```diff
@@ -3,16 +3,19 @@ package main
 
 import "fmt"
 
-var (
-	idx   int     // global variable
-	two   int = 2 // with initialiser
-	three     = 3 // with inferred type
+var j int //single varibale decleration
+
+var ( //declaring multiple variables
+	rat int     // global variable
+	dog int = 2 // with initialiser
+	foo     = 3 // with inferred type
 )
 
 func main() {
-	fmt.Printf("idx  - Val: %v, Type: %T\n", idx, idx)
-	fmt.Printf("two  - Val: %v, Type: %T\n", two, two)
-	fmt.Printf("three- Val: %v, Type: %T\n\n", three, three)
+	fmt.Printf("j- Val: %v, Type: %T\n\n", j, j)
+	fmt.Printf("rat  - Val: %v, Type: %T\n", rat, rat)
+	fmt.Printf("dog  - Val: %v, Type: %T\n", dog, dog)
+	fmt.Printf("foo- Val: %v, Type: %T\n\n", foo, foo)
```

Comment:
These `Printf` statements were previously formatted to all line up on the dash. It now no longer does, so it looks a bit ratty. Either they should all have no space before the dash, or they should all line up.